docs/updates/migrate-v4.md

```diff
@@ -1,39 +1,56 @@
 # Migrate to v4
 
 !!! tip "In Development"
-	**UI5 Tooling V4 is still in development 🚧**
+    **UI5 Tooling 4.0 has been released 🎉**
 
-	Please use UI5 Tooling V3 by installing the latest version via: `npm i --save-dev @ui5/cli@latest`
+    Install the latest version via: `npm i --save-dev @ui5/cli@latest`
 
-	And find the announcement blog post here: **[SAP Community: UI5 Tooling 3.0](https://blogs.sap.com/2023/02/10/ui5-tooling-3.0/)**
+    Find the announcement blog post here: **[SAP Community: UI5 Tooling 4.0](https://blogs.sap.com/)**
+
+## UI5 2.x Compatibility
+
+*Also see the blog post [SAP Community: Introducing OpenUI5 2.x](https://community.sap.com/t5/open-source-blogs/introducing-openui5-2-x/ba-p/13580633)*
+
+UI5 Tooling 4.0 is required for building UI5 2.x projects. The UI5 2.x framework libraries define Specification Version 4.0 and therefore can't be built using older UI5 Tooling versions.
+
+For applications and libraries running with UI5 2.x, the use of Specification Version 4.0 is not enforced. However it is highly recommended since only then UI5 Tooling will ensure UI5 2.x compatibility of the generated bundles.
```

Comment:
```suggestion
For applications and libraries running with UI5 2.x, the use of Specification Version 4.0 is not enforced. It is highly recommended, however, since only then UI5 Tooling will ensure UI5 2.x compatibility of the generated bundles.
```